dev-python/elasticsearch-py/elasticsearch-py-7.14.1.ebuild

```diff
@@ -0,0 +1,185 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{8..10} )
+
+inherit distutils-r1
+
+DESCRIPTION="Official Python low-level client for Elasticsearch"
+HOMEPAGE="https://github.com/elastic/elasticsearch-py"
+SRC_URI="
+	https://github.com/elastic/elasticsearch-py/archive/v${PV}.tar.gz -> ${P}.tar.gz
+	test? (
+		https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-${PV}-no-jdk-linux-x86_64.tar.gz
+	)"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="async doc test"
+
+PROPERTIES="test_network"
+RESTRICT="test"
+
+RDEPEND="
+	dev-python/certifi[${PYTHON_USEDEP}]
+	>=dev-python/urllib3-1.26.0[${PYTHON_USEDEP}]
+	<dev-python/urllib3-2.0.0[${PYTHON_USEDEP}]
+	async? (
+		>=dev-python/aiohttp-3[${PYTHON_USEDEP}]
+		<dev-python/aiohttp-4[${PYTHON_USEDEP}]
+	)"
+DEPEND="
+	test? (
+		${RDEPEND}
+		>=dev-python/requests-2.0.0[${PYTHON_USEDEP}]
+		<dev-python/requests-3.0.0[${PYTHON_USEDEP}]
+		dev-python/mock[${PYTHON_USEDEP}]
+		dev-python/numpy[${PYTHON_USEDEP}]
+		dev-python/pandas[${PYTHON_USEDEP}]
+		dev-python/python-dateutil[${PYTHON_USEDEP}]
+		>=dev-python/pyyaml-5.4[${PYTHON_USEDEP}]
+		virtual/jre:*
+		async? (
+			dev-python/pytest-asyncio[${PYTHON_USEDEP}]
+			dev-python/unasync[${PYTHON_USEDEP}]
+		)
+	)"
+
+distutils_enable_sphinx docs/sphinx dev-python/sphinx_rtd_theme
+distutils_enable_tests pytest
+
+src_prepare() {
+	default_src_prepare
+
+	if use test; then
+		local es_port="25124"
+
+		export ES_DIR="${WORKDIR}/elasticsearch-${PV}"
+		export ES_INSTANCE="gentoo-es-py-test"
+		export ES_JAVA_HOME=$(java-config -g JAVA_HOME || die)
```

Comment:
This gives me the wrong directory, 
`/opt/openjdk-bin-8.302_p08/bin' which is empty. The right one is `/opt/openjdk-bin-8/bin`. Why this happens though, and why openjdk-bin installs empty directories, no idea. Sounds like opendjdk-bin bug. 11.0.12_p7 does not have this problem.

```
Future versions of Elasticsearch will require Java 11; your Java version from [/opt/openjdk-bin-8/jre]    does not meet this requirement. Consider switching to a distribution of Elasticsearch with a bundled JDK. If you are already using a distribution with a bundled JDK, ensure the JAVA_HOME environment variable is  not set.
```
So maybe we should restrict it to >=11